terraform/shared/modules/ecr/ecr.tf

```diff
@@ -32,9 +34,22 @@ resource "aws_ecr_lifecycle_policy" "gdx_data_share_poc" {
         },
         {
             "rulePriority": 2,
+            "description": "Keep last 3 tagged demo images",
+            "selection": {
+                "tagStatus": "tagged",
+                "tagPrefixList": ["dev"],
+                "countType": "imageCountMoreThan",
+                "countNumber": 3
+            },
+            "action": {
+                "type": "expire"
+            }
+        },
+        {
+            "rulePriority": 3,
```

Comment:
Do we need to add prod at some point when we've got it?

Reply:
Yes